_workinggroups/dataanalysis.md

```diff
@@ -24,4 +24,4 @@ well-defined set of problems.
 [Meetings](https://indico.cern.ch/category/10914)
 [Forum](https://groups.google.com/forum/#!forum/hsf-analysis-wg)
 
-Convenors: Paul Laycock (Belle II), Danilo Piparo (CERN EP-SFT), Andrea Rizzi (CMS)
+Convenors: Paul Laycock (Belle II), TJ Khoo (ATLAS), Andrea Rizzi (CMS)
```

Comment:
One could add a section on previous conveners as I did for PyHEP. It may even help down the line the younger colleague wanting to show that he contributed to the HSF.

Reply:
This same comment goes for the other WGs, of course.

Reply:
Yep, done - thanks for that, it's a good idea